data/contact/3dbb59fc-905c-4708-a4f2-e30bc9ac2903.yml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
confusing that we'd load some (GIST) c2's via YML and others (HTTP) without it..

Reply:
I agree. Updating now.